cross-project-tests/dtlto/README.txt

```diff
@@ -0,0 +1,2 @@
+                                                                   -*- rst -*-
```

Comment:
Why not just use README.md or README.rst?

Reply:
Yes, a `.md` makes sense. I have also added another sentence explaining why the tests are in this area.